Jenkinsfile

```
@@ -28,51 +27,18 @@ pipeline {
       }
     }
 
-    stage('NPM Install') {
-      agent {
-        docker {
-          image 'node:16.13.1'
-          reuseNode true
-        }
-      }
-      steps {
-        sh 'yarn install'
-      }
-    }
-
-    stage('Build Production') {
+    stage('Lint and test') {
       agent {
         docker {
           image 'node:16.13.1'
           reuseNode true
         }
       }
       steps {
+        sh 'npm install'
```

```suggestion
        sh 'npm clean-install'
```

Using `npm install` is considered a bad practice as it will try to update non locked dependencies, while clean-install reads from the lock file.

Not a blocker, but better practice (and additional safety nets when using old versions like here)